tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -1125,6 +1124,73 @@ class WskBasicUsageTests extends TestHelpers with WskTestHelpers {
     }
   }
 
+  it should "invoke a feed action with the correct lifecyle event when creating a feed trigger" in withAssetCleaner(
```

Comment:
@rabbah, what are your thoughts on not using the `assetCleaner` to rolling these three tests into one using something like the below.
```
  it should "invoke a feed action with the correct lifecyle event when creating, reading, and deleting a feed trigger" in withAssetCleaner(
    wskprops) { (wp, assetHelper) =>
    val actionName = "echo"
    val triggerName = "feedTest"

    assetHelper.withCleaner(wsk.action, actionName) { (action, _) =>
      action.create(actionName, Some(TestUtils.getTestActionFilename("echo.js")))
    }

    try {
      wsk.trigger.create(triggerName, feed = Some(actionName)).
          stdout should include(""""lifecycleEvent": "CREATE"""")

      wsk.trigger.get(triggerName).
          stdout should include(""""lifecycleEvent": "READ"""")
    } finally {
      wsk.trigger.delete(triggerName).
          stdout should include(""""lifecycleEvent": "DELETE"""")
    }
  }
```

Reply:
the asset cleaner is a convenience - use it where it make sense.
your code looks good to me.